app/templates/partials/guidance.html

```diff
@@ -0,0 +1,17 @@
+{% import 'macros/helpers.html' as helpers %}
+
+{% for item in guidance_content.content %}
+  {%- if item.title -%}
+    <h3 class="venus">{{item.title}}</h3>
```

Comment:
Need confirmation that these should be h tags at these levels, has @hamishtaplin or @bennpearson been involved?